junifer/datagrabber/hcp1200/datalad_hcp1200.py

```diff
@@ -23,50 +25,37 @@ class DataladHCP1200(DataladDataGrabber, HCP1200):
 
     Parameters
     ----------
-    datadir : str or Path or None, optional
-        The directory where the datalad dataset will be cloned. If None,
-        the datalad dataset will be cloned into a temporary directory
-        (default None).
-    tasks : {"REST1", "REST2", "SOCIAL", "WM", "RELATIONAL", "EMOTION", \
-            "LANGUAGE", "GAMBLING", "MOTOR"} or list of the options or None \
-            , optional
-        HCP task sessions. If None, all available task sessions are selected
-        (default None).
-    phase_encodings : {"LR", "RL"} or list of the options or None, optional
-        HCP phase encoding directions. If None, both will be used
-        (default None).
+    types : list of {``DataType.BOLD``, ``DataType.T1w``, ``DataType.Warp``}, \
+            optional
+        The data type(s) to grab.
+    datadir : pathlib.Path, optional
+        That path where the datalad dataset will be cloned.
+        If not specified, the datalad dataset will be cloned into a temporary
+        directory.
+    tasks : list of :enum:`.HCP1200Task`, optional
```

Comment:
will this expand? Otherwise I prefer the previous docstring as it tells you exactly what your options are.